packages/image-block/src/ImageBlock.tsx

```diff
@@ -56,6 +57,10 @@ export const ImageBlock = withAttachmentsProvider(({ appBridge }: BlockProps) =>
     const { blockAssets, deleteAssetIdsFromKey, updateAssetIdsFromKey } = useBlockAssets(appBridge);
     const image = blockAssets?.[IMAGE_ID]?.[0];
     const attachmentCount = blockAssets[ATTACHMENTS_ASSET_ID]?.length || 0;
+    const isImageDownloadable =
+        image?.isDownloadProtected === false &&
+        isDownloadable(blockSettings.security, blockSettings.downloadable, assetDownloadEnabled);
```

Comment:
I think it would make sense to move that into the helpers and add some unit tests for it

Reply:
And as i just saw it the helper that is already there, should move into the `shared`

Reply:
Updated, and added some tests to test the new logic, as the isDownloadable is already tested in `guideline-blocks-settings`